grades/models.py

```diff
@@ -0,0 +1,49 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+class GradingSystem(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(default='About the grading system!')
+    can_be_used = models.BooleanField(default=False)
```

Comment:
Actually in PR description I have mentioned that fields ```can_be_used``` and ```order``` in models are for future use.
I had planned few things with these fields. These two are not used anywhere currently.
Will currently remove it in my next commit to avoid confusion.